pkg/registration/product.go

```diff
@@ -1,12 +1,19 @@
 package registration
 
+import (
+	"encoding/json"
+
+	"github.com/SUSE/connect-ng/pkg/connection"
+)
+
 // Product as defined from SCC'S API.
 type Product struct {
-	Name    string `json:"identifier"`
-	Version string `json:"version"`
-	Arch    string `json:"arch"`
-	Summary string `json:"summary,omitempty"`
-	IsBase  bool   `json:"isbase"`
+	Name       string `json:"name"`
```

Comment:
I changed `Name` to `name`. I was wondering what did we think about the `LongName`, `FriendlyName` topic?
I added `identifier` to make things more clear.

@mssola do you remember?

Reply:
Identifier should be better. iirc there was a mismatch on how "names" were named on SCC and how they were named here in SUSEConnect. This is a good chance to come up with a clean slate, so let's use names like `Identifier` which sound more to the point.

I'd still include `FriendlyName` as it is used by `zypper-migration`. As for `LongName`, I'm not sure, and I see that we do use it internally. Hence, I'd still keep it but remove it if it doesn't make sense whenever we use this public library from within SUSEConnect.

Reply:
I removed `LongName`. `FriendlyName`, `Name` and `Identifier` exists now as in the API